docs/Parameters.rst

```diff
@@ -1010,6 +1010,8 @@ Metric Parameters
 
       -  ``auc``, `AUC <https://en.wikipedia.org/wiki/Receiver_operating_characteristic#Area_under_the_curve>`__
 
+      -  ``ap``, `average precision score <https://scikit-learn.org/stable/modules/generated/sklearn.metrics.average_precision_score.html>`__
```

Comment:
could this be called `average_precision` instead of `ap`? I think the very short names can be confusing unless they're very common acronyms like AUC.

Reply:
Done :)

Reply:
awesome, thanks! I think it will help